tests/Http/SavedSearches/ReadSavedSearchesRequestHandlerTest.php

```diff
@@ -42,21 +43,23 @@ public function it_gets_saved_searches_of_current_user(): void
             ),
             new SavedSearch(
                 new StringLiteral('Saved search 1'),
-                new QueryString('city:herent')
+                new QueryString('city:herent'),
+                new StringLiteral('b706ca05-9139-422c-92e4-8aeb512466d6')
             ),
         ];
 
         $this->savedSearchRepository->expects($this->once())
             ->method('ownedByCurrentUser')
             ->willReturn($savedSearches);
 
-        $actualResponse = $this->readSavedSearchesController->ownedByCurrentUser();
+        $readSavedSearchesRequest = $this->psr7RequestBuilder
+            ->build('GET');
+
+        $response = $this->readSavedSearchesRequestHandler->handle($readSavedSearchesRequest);
 
         $this->assertEquals(
```

Comment:
I would check the full response here, so we also take into account the status code.